### PR TITLE
Being honest about the state of call caching

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Comment by cjllanwarne** [comment]
Let's reply with honesty rather than sending an unrecognised message to an actor which probably has nothing to do with call caching anymore

**Comment by mcovarr** [comment]
:+1: it really is the best policy

File: engine/src/main/scala/cromwell/webservice/CromwellApiService.scala [hunk 1]
```diff
@@ -182,12 +182,8 @@ trait CromwellApiService extends HttpService with PerRequestCreator with Service
...
         val queryParameters = parameters map { case (k, v) => QueryParameter(k, v) }
         post {
-          Try(WorkflowId.fromString(workflowId)) match {
-            case Success(w) =>
-              requestContext =>
-                perRequest(requestContext, CromwellApiHandler.props(workflowManager), CromwellApiHandler.ApiHandlerCallCaching(w, queryParameters, callFqn))
-            case Failure(_) => invalidWorkflowId(workflowId)
-          }
+          // TODO: PBE: Certainly want to do something for this! But probably not to the WMA
+          failBadRequest(new RuntimeException(s"Call caching is currently unsupported."))
```
Comment (geoffjentry) [inline engine/src/main/scala/cromwell/webservice/CromwellApiService.scala]: `UnsupportedOperationException`?

**Comment by geoffjentry** [comment]
👍

**Pull request merged**